scalopus_tracing/test/test_native_tracepoints.cpp

```diff
@@ -131,8 +131,6 @@ int main(int /* argc */, char** /* argv */)
     {
       TRACE_SCOPE_RAII("different_thread");
     }
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
-    // prevent this thread from being cleaned up before the tracepoint goes from the server to the client.
```

Comment:
Looks like I already envisioned the problem we found in #18, but back then I didn't solve it, since I expect this to be rare.